static/js/components/ShareDataForm.jsx

```diff
@@ -0,0 +1,238 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { useSelector, useDispatch } from "react-redux";
+import MUIDataTable from "mui-datatables";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import { useForm, Controller } from "react-hook-form";
+import Autocomplete from "@material-ui/lab/Autocomplete";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+
+import { showNotification } from "baselayer/components/Notifications";
+
+import FormValidationError from "./FormValidationError";
+import Plot from './Plot';
+
+import * as photometryActions from "../ducks/photometry";
+import * as spectraActions from "../ducks/spectra";
+import * as sourceActions from "../ducks/source";
+import styles from "./Source.css";
+
+
+const createPhotRow = (id, mjd, mag, magerr, limiting_mag, instrument, filter, groups) => (
+  {
+    id,
+    mjd: Number(mjd).toFixed(3),
+    mag: mag === null ? null : Number(mag).toFixed(4),
+    magerr: magerr === null ? null : Number(magerr).toFixed(4),
+    limiting_mag: Number(limiting_mag).toFixed(4),
+    instrument,
+    filter,
+    groups
+  }
+);
+
+const createSpecRow = (id, instrument, observed, groups) => (
+  {
+    id,
+    instrument,
+    observed,
+    groups
+  }
+);
+
+const photHeadCells = [
+  { name: "id", label: "ID" },
+  { name: "mjd", label: "MJD" },
+  { name: "mag", label: "Mag" },
+  { name: "magerr", label: "Mag Error" },
+  { name: "limiting_mag", label: "Limiting Mag" },
+  { name: "instrument", label: "Instrument" },
+  { name: "filter", label: "Filter" },
+  { name: "groups", label: "Existing Groups" },
+];
```

Comment:
```suggestion
{ name: "owner", label: "Owner" }
];
```

Reply:
@dannygoldstein we don't store owner/uploader info yet...